lib/kafkalib/topic.go

```diff
@@ -47,6 +47,7 @@ type TopicConfig struct {
 	IncludeArtieUpdatedAt     bool                        `yaml:"includeArtieUpdatedAt"`
 	IncludeDatabaseUpdatedAt  bool                        `yaml:"includeDatabaseUpdatedAt"`
 	BigQueryPartitionSettings *partition.BigQuerySettings `yaml:"bigQueryPartitionSettings,omitempty"`
+	AdditionalMergeColumns    []string                    `yaml:"additionalMergeColumns,omitempty"`
```

Comment:
When we go to Prod, we should create a separate type for this so we can embed additional information